Vagrantfile

```diff
@@ -77,23 +88,14 @@ Vagrant.configure("2") do |config|
 
   config.vm.provision "docker" do |d|
     d.pull_images "ailispaw/dnsdock:1.16.4"
+    d.post_install_provision "shell", inline:"docker rm -fv dnsdock"
```

Comment:
This will fail when there is no `dnsdock`

Reply:
Ignored on failure with `|| true`